2014 Primary Election Amador test failure

Missing write-in candidate. Not available at precinct-level, but just totals on the last page of the [Statement of Votes Cast](http://www.co.amador.ca.us/home/showdocument?id=19064)

```
E           AssertionError: Amador failed
E           assert {('"Bo" Bogda...'): 1708, ...} == {('"Bo" Bogdan...'): 1708, ...}
E             Omitting 65 identical items, use -v to show
E             Left contains more items:
E             {('Patrick D. Hogan', 'Amador', 5.0, 'State Assembly'): 5}
E             Use -v to get the full diff
```

--- a/2014/20140603__ca__primary__amador__precinct.csv
+++ b/2014/20140603__ca__primary__amador__precinct.csv
@@ -3462,3 +3462,4 @@ Amador,CP83,Treasurer,,DEM,John Chiang,11
 Amador,CP83,U.S. House,4,REP,"Arthur ""Art"" Moore",4
 Amador,CP83,U.S. House,4,IND,Jeffrey D. Gerlach,6
 Amador,CP83,U.S. House,4,REP,Tom McClintock,10
+Amador,Write-In,State Assembly,5,LIB,Patrick D. Hogan,5

--- a/src/swdb/writeins.py
+++ b/src/swdb/writeins.py
@@ -20,4 +20,15 @@ def alameda_p14(candidate_norm):
              'precinct': 'Write-In'} for
             c, v in cand_split if v != 0]
 
-WRITEINS = {'P14': {'Alameda': alameda_p14}}
+
+def amador_p14(_):
+    return [{'candidate': 'Patrick D. Hogan',
+             'county': 'Amador',
+             'office': 'State Assembly',
+             'district': '5',
+             'party': 'LIB',
+             'precinct': 'Write-In',
+             'votes': 5}]
+
+WRITEINS = {'P14': {'Alameda': alameda_p14,
+                    'Amador':  amador_p14}}

